types/client.ts

```diff
@@ -46,3 +46,26 @@ export interface TeamProfile {
 	devpost: URL;
 	members: string[];
 }
+
+export interface Report {
+	_id?: string;
+	email: string;
+	name: string;
+	role: ReportRole;
+	description: string;
+	date: string;
+	ghIssueNumber: number;
+	ghAssignee: string;
+	ghUrl: string;
+}
+
+export enum ReportRole {
+	HACKER = 'HACKER',
+	JUDGE = 'JUDGE',
+	ORGANIZER = 'ORGANIZER',
```

Comment:
im actually surprised we don't already have an interface for HACKER, JUDGE, ORGANIZER... damn.